features/Skylighting/Shaders/Skylighting/SkylightingBlurCS.hlsl

```diff
@@ -0,0 +1,73 @@
+#include "../Common/FrameBuffer.hlsl"
+#include "../Common/GBuffer.hlsli"
+#include "../Common/VR.hlsli"
+
+Texture2D<half> DepthTexture : register(t0);
+
+Texture2D<half4> SkylightingTexture : register(t1);
+RWTexture2D<float4> SkylightingTextureRW : register(u0);
+
+cbuffer PerFrame : register(b0)
```

Comment:
Maybe move out to shared file between `SkylightingBlurCS.hlsl` and `SkylightingCS.hlsl`? Also `GetScreenDepth` method is duplicated

Reply:
this is getting replace anyway so will leave for now